test/claim/claim-with-request.testSuite.ts

```diff
@@ -286,4 +289,80 @@ export const claimWithRequestTestSuite = () => {
       requestOrigin
     );
   });
+  it(`should fetch all issued claims by revoker did`, async () => {
```

Comment:
```suggestion

  it(`should fetch all issued claims that revoker is allowed to revoke`, async () => {
```
maybe that name will be better?